templates/js-standby/.actor/actor.json

```
@@ -0,0 +1,12 @@
+{
+    "actorSpecification": 1,
+    "name": "js-standby",
+    "title": "Standby Actor in JavaScript",
+    "description": "Standby Actor in JavaScript.",
+    "version": "0.0",
+    "buildTag": "latest",
+    "meta": {
```

Let's add the `usesStandbyMode` top level property as per discussion in Slack. Also to all the other templates.
```suggestion
    "usesStandbyMode": true,
    "meta": {
```